src/facts/black-eyed-susan.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Black-eyed Susan
+image:
+  src: ../images/black-eyed-susan.jpg
+  url: https://www.flickr.com/photos/stanzim
```

Comment:
Please link to the specific image

Reply:
Sorry Paul I thought you linked it to the user's profile last time. I will get the link right next time.